src/sentry/api/serializers/rest_framework/project.py

```diff
@@ -9,16 +9,22 @@
 
 @extend_schema_field(field=OpenApiTypes.STR)
 class ProjectField(serializers.Field):
-    def __init__(self, scope="project:write"):
+    def __init__(self, scope="project:write", id_allowed=False):
```

Comment:
we should eventually support id lookup everywhere this field is used, but didn't want to pollute this PR with the implications of that